datumaro/cli/commands/diff.py

```diff
@@ -202,8 +203,9 @@ def diff_command(args):
         output_file = osp.join(dst_dir,
             generate_next_file_name('diff', ext='.json', basedir=dst_dir))
         log.info("Saving diff to '%s'" % output_file)
-        with open(output_file, 'w') as f:
-            json.dump(output, f, indent=4, sort_keys=True)
+        with open(output_file, 'wb') as f:
+            f.write(orjson.dumps(output,
+                option=orjson.OPT_SORT_KEYS | orjson.OPT_INDENT_2))
```

Comment:
IMO, we need to add utility functions for common operations (i.e. "save JSON to file" and "load JSON from file") and replace all these chunks of code with calls to those functions. That way:

1. We eliminate repetitive code.
2. We'll be able to easily swap out the JSON implementation later if we need to.

Reply:
Done